packit_service/worker/jobs.py

```diff
@@ -103,12 +113,23 @@ def get_config_for_handler_kls(
     handler_kls: Type[JobHandler], event: Event, package_config: PackageConfig
 ) -> List[JobConfig]:
     """
-    Get a JobConfig relevant to event and the handler class.
+    Get a list of JobConfigs relevant to event and the handler class.
+
+    We need to find all job configurations that:
+    - can be run by the given handler class
+    - that matches the trigger of the event
```

Comment:
Make it explicit that this is 'or':

```suggestion
    We need to find all job configurations that either:
    - can be run by the given handler class OR
    - that matches the trigger of the event
```